Core/CookieStorage.swift

```diff
@@ -57,28 +57,4 @@ public class CookieStorage {
         os_log("cleared cookies", log: generalLog, type: .debug)
     }
 
-    func setCookie(_ cookie: HTTPCookie) {
-        os_log("storing cookie %s %s %s", log: generalLog, type: .debug, cookie.domain, cookie.name, cookie.value)
-        
-        var cookieData = [String: Any?]()
-        cookie.properties?.forEach({
-            cookieData[$0.key.rawValue] = $0.value
-        })
-
-        // Same site policy is not returned in the cookie properties so have to use iOS 13 api to get it - setting is fine.  Leaving same site policy
-        //  untouched for previous versions of iOS as changing from Strict to Lax blindly could be a security issue if sites have set it as such.
-        //  See https://www.owasp.org/index.php/SameSite
-        if #available(iOS 13, *) {
-            cookieData[HTTPCookiePropertyKey.sameSitePolicy.rawValue] = cookie.sameSitePolicy ?? HTTPCookieStringPolicy.sameSiteLax
```

Comment:
So ... another problem here is that we make this change deliberately.  It's not quite the right logic though, so I'm just getting confirmation we actually need it

cc @kdzwinel

Reply:
I talked to Konrad about this and we're all good given that webkit blocks 3p cookies anyway (so long as the user hasn't disabled that, which we can't do much about)